### ISSUE TITLE
Can't change volume from remote

### ISSUE THREAD

**Issue by svenstaro** [plexinc/plex-media-player#327]
I'm using the official Android plex app to remote control my PMP. If I tap the audio icon in the remote control on Android, I get a popup saying "Volume not supported". How can I enable this so that I can change my volume with my remote?

**Comment by danjames92** [comment]
Are you passing through the audio codecs? If so, it's not surprising.

**Comment by svenstaro** [comment]
How would I check that?

**Comment by svenstaro** [comment]
Ok frankly, PMP should definitely be able to change the volume even if the server isn't re-encoding the files. That just doesn't make sense. Any program can set its own output volume. mpv which PMP is using also can easily set the output volume without re-encoding the media.

**Comment by wm4** [comment]
Nothing can change the volume when passing through compressed data. That's just how it works. You can only change the volume on the receiver.

**Comment by svenstaro** [comment]
Well that's the whole idea here, I think. I would expect the volume controls to be wired to mpv's own. mpv, as I understand, is the receiver in this case.

**Comment by wm4** [comment]
No, if actual passthrough is used, the thing on the other end of the HDMI connection is the receiver. Only whatever actually decodes the audio can change its volume.

**Comment by atrus05** [comment]
You guys are talking about different things. Passthrough and transcoding are two wildly different things. 
@svenstaro: uncheck the DTS and AC3 checkboxes in PMP Setttings. If it then starts to work then you had passthrough enabled.

**Comment by danjames92** [comment]
It also will help for you to change it so the audio codec appears on your AVR so you can see either PCM (passthrough) or the codec (DTS-HD MA etc) bitstream.

**Comment by svenstaro** [comment]
If I go to <account name in upper right corner> -> Settings -> Audio it doesn't show any of those settings. When I tick "Show Advanced Settings", nothing happens. My settings are like this: Basic, 5.1, Autoselect device, Normalize Downmixed Volume, Show Advanced Settings. That's it.

**Comment by danjames92** [comment]
That's why. Pretty sure only the openelec version of Linux is supported AFAIK.

**Comment by svenstaro** [comment]
Well PulseAudio is pretty common. Is support for this planned? It should be
quite easy considering that qt5 is used (although I don't know how exactly
PMP uses it in this project). I'd actually be willing to contribute of
someone gave me some instructions on how to do this particular thing so
that it may be merged.

**Comment by jgilfoil** [comment]
I'm also unable to use the android remote to control the volume on PMP, however there are two differences. I'm on windows 10 and i don't get an error. The volume control just has no affect on the audio level. The only way to affect the audio in PMP is to hit the '+' or '-' keys on the keyboard.

The remote volume control works just fine in Firefox on the same windows 10 machine. 

Should this be a separate bug report?

**Comment by wm4** [comment]
> I'm on windows 10 and i don't get an error. The volume control just has no affect on the audio level. 

Is it controlling the system audio volume? Does disabling "Exclusive Mode" in the PMP audio settings help?

**Comment by jgilfoil** [comment]
> Is it controlling the system audio volume? Does disabling "Exclusive Mode" in the PMP audio settings help?

I tried it both ways with exclusive mode on and off. Neither worked. After playing with it more I also realized that attempting to change the volume with the remote makes the app lose control of the playback... In other words:

Reproduction steps:
1. Play a video in plex media player
2. From android app connect to PMP device.
3. Attempt to change volume with android volume keys
4. can't pause/stop/rewind or fast foward, but the time counter keeps working. 
5. Reconnect the app to PMP device to regain control.
